ngi_pipeline/engines/piper_ngi/launchers.py

```diff
@@ -210,10 +210,12 @@ def sbatch_piper_sample(command_line_list, workflow_name, project, sample,
     # Paths to the various data directories
     project_dirname = project.dirname
     sample_dirname = sample.dirname
-    perm_analysis_dir = os.path.join(project.base_path, "ANALYSIS", project_dirname)
-    scratch_analysis_dir = os.path.join("$SNIC_TMP/ANALYSIS/", project_dirname)
+    perm_analysis_dir = os.path.join(project.base_path, "ANALYSIS", project_dirname, "piper_ngi")
+    scratch_analysis_dir = os.path.join("$SNIC_TMP/ANALYSIS/", project_dirname, "piper_ngi")
```

Comment:
is this taking an env variable from the shell? I didn't know that you can do that